docs/getting-started/integrations/react-multiroot.md

```diff
@@ -28,22 +26,21 @@ The easiest way to use multi-root CKEditor&nbsp;5 in your React application is b
 
 ## Quick start
 
-This guide assumes you already have a React project. If you want to create a new one, you can use the [`create-react-app`](https://create-react-app.dev/) CLI. It allows you to create and customize your project with templates. For example, you can set up your project with TypeScript support.
+This guide assumes you already have a React project. If you want to create a new one, you can use the [Vite](https://vitejs.dev/guide/) CLI. It allows you to create and customize your project with templates. For example, you can set up your project with TypeScript support.
 
-Install the [CKEditor&nbsp;5 WYSIWYG editor package for React](https://www.npmjs.com/package/@ckeditor/ckeditor5-react) and the {@link getting-started/legacy-getting-started/predefined-builds#multi-root-editor multi-root editor build}. Assuming that you picked [`@ckeditor/ckeditor5-build-multi-root`](https://www.npmjs.com/package/@ckeditor/ckeditor5-build-multi-root):
+Install the [CKEditor&nbsp;5 WYSIWYG editor package for React](https://www.npmjs.com/package/@ckeditor/ckeditor5-react) and the {@link getting-started/legacy-getting-started/predefined-builds#multi-root-editor multi-root editor type}. Assuming that you picked the Multi Root Editor:
 
 ```bash
-npm install --save @ckeditor/ckeditor5-react @ckeditor/ckeditor5-build-multi-root
+npm install ckeditor5 @ckeditor/ckeditor5-react
 ```
 
 Use the `useMultiRootEditor` hook inside your project:
 
 ```tsx
 // App.jsx / App.tsx
 
-import React from 'react';
+import { MultiRootEditor } from 'ckeditor5';
 import { useMultiRootEditor } from '@ckeditor/ckeditor5-react';
-import MultiRootEditor from '@ckeditor/ckeditor5-build-multi-root';
 
 const App = () => {
 	const editorProps = {
```

Comment:
I miss some standard plugins here. I'm not sure if this editor will run, and allow writing anything?

Reply:
The snippet wasn't working from the start, so I assumed it was just for demo purposes. But I can make it functional - I was considering that.